commit e333b32b09351ae31b0bb56040075d4c72a72a1f
Author: Knicko <knicko@knicko-EP35-DS4>
Date:   Tue Oct 25 00:03:37 2016 +0200

    Revert "system/helpers: allow to build with c++11 compilers"
    
    This reverts commit ae345d11d30779acb9877643919507d1bcb07c9a.

diff --git a/src/system/helpers.cpp b/src/system/helpers.cpp
index 380f2f8..dbe34b0 100644
--- a/src/system/helpers.cpp
+++ b/src/system/helpers.cpp
@@ -1254,7 +1254,6 @@ std::vector<std::string> split(const std::string &s, char delim)
        return vec;
 }
 
-#if __cplusplus <= 201103L
 std::string to_string(int i)
 {
        std::stringstream s;
@@ -1296,7 +1295,6 @@ std::string to_string(unsigned long long i)
        s << i;
        return s.str();
 }
-#endif
 
 /**
  * C++ version 0.4 std::string style "itoa":
diff --git a/src/system/helpers.h b/src/system/helpers.h
index a18f495..0508aeb 100644
--- a/src/system/helpers.h
+++ b/src/system/helpers.h
@@ -119,17 +119,12 @@ class CFileHelpers
 
 uint32_t GetWidth4FB_HW_ACC(const uint32_t _x, const uint32_t _w, const bool max=true);
 
-#if __cplusplus <= 201103L
 std::string to_string(int);
 std::string to_string(unsigned int);
 std::string to_string(long);
 std::string to_string(unsigned long);
 std::string to_string(long long);
 std::string to_string(unsigned long long);
-#else
-/* hack... */
-#define to_string(x) std::to_string(x)
-#endif
 
 std::string itoa(int value, int base);

